src/esp/metadata/managers/AttributesManagerBase.h

```diff
@@ -46,7 +46,7 @@ class AttributesManager : public esp::core::ManagedContainer<T, Access> {
   AttributesManager(const std::string& attrType, const std::string& JSONTypeExt)
       : esp::core::ManagedContainer<T, Access>::ManagedContainer(attrType),
         JSONTypeExt_(JSONTypeExt) {}
-  virtual ~AttributesManager() = default;
+  ~AttributesManager() override = default;
```

Comment:
Overriding destructors.  Had to research this one, but is a good change that I know I would have never thought of.